examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -47,6 +48,41 @@ using namespace ::chip::DeviceLayer;
 uint32_t identifyTimerCount;
 constexpr uint32_t kIdentifyTimerDelayMS = 250;
 
+namespace {
+
+void StartMdnsAdvertise(bool isIpv4)
```

Comment:
what does isIpv4 do? it seems its only use is: if ipv4 is not enabled and this is true, we are a noop.

Could we name it somehow else or do start no matter what and only control 'CONFIG_ENABLE_IPV4' ?